servers/src/main/java/tachyon/master/InodeFile.java

```diff
@@ -54,8 +62,12 @@ static InodeFile loadImage(ImageElement ele) throws IOException {
     final String ufsPath = ele.getString("ufsPath");
     final int dependencyId = ele.getInt("depId");
     final long lastModificationTimeMs = ele.getLong("lastModificationTimeMs");
+    final String username = ele.getString("username");
+    final String groupname = ele.getString("groupname");
+    final short permission = ele.getShort("permission");
 
```

Comment:
Question: what if those values are not present in the image file? For example, when people update Tachyon into this version?